spec/views/layouts/_login.html.haml_spec.rb

```diff
@@ -1,82 +0,0 @@
-require "rails_helper"
```

Comment:
This test is no longer needed because this modal is gone!

Reply:
Seems we needed that test to keep our coverage up though :D

```
Coverage report generated for RSpec to /home/runner/work/swapmyvote/swapmyvote/coverage. 916 / 1316 LOC (69.6%) covered.
Line coverage (69.60%) is below the expected minimum coverage (70.00%).
Branch coverage ([56](https://github.com/swapmyvote/swapmyvote/actions/runs/9713695862/job/26811036363?pr=897#step:8:57).84%) is below the expected minimum coverage (58.00%).
SimpleCov failed with exit 2 due to a coverage related error
```

Reply:
I was right, the relevant commit, for code coverage, says "set coverage thresholds so it only just passes".

@stevebaxter please adjust the thresholds a little bit in spec/spec_helper.rb.